### PR TITLE
1128.01 - Fix Broken Links (Getting Started): Update installation guide.

### PR DESCRIPTION
**Pull request by philleepflorence**
Initial docs PR...

### CODE REVIEW

File: docs/guides/installation/cli.md [hunk 1]
```diff
@@ -28,12 +32,22 @@ Variants In addition to the databases above, other variants are also supported,
...
+Navigate to the directory where you wish to create a new Directus project.
+The new project and directory will be created inside the current directory.
 Create a new Directus project by running the following npm command.
 
+<!-- prettier-ignore-start -->
+::: warning
+To avoid installation issues, do not name your project _directus_.
+:::
+<!-- prettier-ignore-end -->
+
```
Comment (rijkvanzanten) [inline docs/guides/installation/cli.md:44]: Why is this the case? What kind of installation issues would you run into when naming your project `directus`?

Comment (philleepflorence) [reply docs/guides/installation/cli.md:44]: I got several errors while installingand it hung without completing...I can try again and send the errors to you?

Comment (rijkvanzanten) [reply docs/guides/installation/cli.md:44]: Ohhhh I think I might know what's the cause there. I think the `npm init` call in the folder sets the name of "your project" to `directus`, into which you're trying to install `directus`

Comment (rijkvanzanten) [reply docs/guides/installation/cli.md:44]: The same would happen if you called your folder something like "rimraf" or any of the other dependency s

Comment (rijkvanzanten) [reply docs/guides/installation/cli.md:44]: I think we should update the CLI so it "scoped" the package name so this name conflict doesn't happen. We could probably prefix the "package name" (so not the folder name) with `directus-project-`

Comment (philleepflorence) [reply docs/guides/installation/cli.md:44]: Yes, you are correct...
**Did you name your project the same as the dependency you're installing?**

Here is the full error:
```
Installing Directus(node:5233) UnhandledPromiseRejectionWarning: Error: Command failed with exit code 1: npm install directus --production --no-optional
npm ERR! code ENOSELF
npm ERR! Refusing to install package with name "directus" under a package
npm ERR! also called "directus". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR! 
npm ERR! For more information, see:
npm ERR!     <https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm>

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/philleepflorence/.npm/_logs/2021-01-08T17_46_02_491Z-debug.log
    at makeError (/Users/philleepflorence/.npm/_npx/5233/lib/node_modules/create-directus-project/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/Users/philleepflorence/.npm/_npx/5233/lib/node_modules/create-directus-project/node_modules/execa/index.js:114:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:5233) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5233) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Comment (rijkvanzanten) [reply docs/guides/installation/cli.md:44]: Lets take it out of the docs (cause I'd like you to be able to name your folder whatever), and I'll open a ticket to get that resolved in the CLI 👍🏻

Comment (philleepflorence) [reply docs/guides/installation/cli.md:44]: Will do...

Comment (philleepflorence) [reply docs/guides/installation/cli.md]: Done!

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**